apps/mobile-alphabet/tsconfig.app.json

```diff
@@ -5,5 +5,5 @@
         "types": ["node"]
     },
     "exclude": ["jest.config.ts", "src/**/*.spec.ts", "src/**/*.spec.tsx", "test-setup.ts"],
-    "include": ["src/**/*.ts", "src/**/*.tsx", "src/**/*.js", "src/**/*.jsx"]
+    "include": ["src/**/*.ts", "src/**/*.tsx", "src/**/*.js", "src/**/*.jsx", "src/**/*.json"]
```

Comment:
Oh do we really want every json file in the build? Can you just explicitly include the config only if that's what you need?